server/server.go

```diff
@@ -49,21 +58,40 @@ func New(cfg Config) (*Server, error) {
 		router = http.NewServeMux()
 	}
 
+	reg := prometheus.NewRegistry()
+	requestDuration := promauto.With(reg).NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "cortex",
+			Name:      "request_duration_seconds",
+			Help:      "Time (in seconds) spent serving HTTP requests.",
```

Comment:
```suggestion
			Help:      "Time (in seconds) spent serving HTTP requests.",
			Buckets:   []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 25, 50, 100},			
```
Let's use the same buckets cortex uses